packages/account/src/providers/provider.ts

```diff
@@ -773,48 +782,56 @@ export default class Provider {
       estimatePredicates = true,
       resourcesOwner,
     }: TransactionCostParams = {}
-  ): Promise<TransactionCost> {
-    const transactionRequest = transactionRequestify(clone(transactionRequestLike));
+  ): Promise<
+    TransactionCost & {
+      estimatedInputs: TransactionRequest['inputs'];
+      estimatedOutputs: TransactionRequest['outputs'];
+    }
+  > {
+    const txRequestClone = clone(transactionRequestify(transactionRequestLike));
     const chainInfo = this.getChain();
     const { gasPriceFactor, minGasPrice, maxGasPerTx } = this.getGasConfig();
-    const gasPrice = max(transactionRequest.gasPrice, minGasPrice);
-    const isScriptTransaction = transactionRequest.type === TransactionType.Script;
+    const gasPrice = max(txRequestClone.gasPrice, minGasPrice);
+    const isScriptTransaction = txRequestClone.type === TransactionType.Script;
+
+    /**
+     * Fund with fake UTXOs to avoid not enough funds error
+     */
```

Comment:
Nit, mixing comment styles

```suggestion
     // Fund with fake UTXOs to avoid not enough funds error
```